Test/client_test.py

```diff
@@ -38,6 +38,16 @@ def test_get_org(self):
         response = self.c.get_org("a_myOrg84")
         self.assertEqual("a_myOrg84",response['org_name'])
 
+    def test_get_build_logs(self):
+        client = Algorithmia.client(api_key=os.environ.get('ALGORITHMIA_API_KEY'))
+        user = os.environ.get('ALGO_USER_NAME')
+        algo = "Echo"
+        result = client.get_build_logs(user,algo)
```

Comment:
Yeah if this could be wrapped around an Algorithm entity like so:
client.algo("user/pass").getBuilds() would be the preferred library approach